docs/_documentation/advanced/viewmodel-lifecycle.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: documentation
 title: ViewModel Lifecycle
-category: Fundamentals
-order: 7
+category: Advanced
```

Comment:
I think this doc should remain on Fundamentals, it's kind of important to know how ViewModels work when starting to use the fwk